core/src/main/java/org/arquillian/smart/testing/hub/storage/local/DuringExecutionLocalStorage.java

```diff
@@ -0,0 +1,44 @@
+package org.arquillian.smart.testing.hub.storage.local;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+public class DuringExecutionLocalStorage extends AfterExecutionLocalStorage {
+
+    public static final String SMART_TESTING_WORKING_DIRECTORY_NAME = ".smart-testing";
+    public static final String TEMPORARY_SUBDIRECTORY = "temporary";
```

Comment:
Let's move it to `Configuration` class as this will be anyway reworked to file-based solution. Having it centralized it will let us move to the file-based approach easier and not forget important bits.

Reply:
Personally, into the `Configuration` class I would put those constants that are used there (or parameters that can be configured or constants containing defaults values and similar). 
If you want to make the constants centralized, then let's create `Constants` class that will contain everything.
These directory names are static values and cannot be configured anyhow.

Reply:
Right, this is something we don't need to expose to end-users. Fine with me.